sale_layout_category_hide_detail/views/account_invoice_view.xml

```diff
@@ -35,28 +37,43 @@
                                     'icon_false': 'fa-eye-slash',
                                 },
                                 'terminology': {
-                                    'hover_true': 'Switch to: details hidden',
-                                    'hover_false': 'Switch to: details shown',
+                                    'hover_true': 'Show details',
+                                    'hover_false': 'Show details',
                                 },
                            }"
                            attrs="{'invisible': [('display_type', '!=', 'line_section')]}"/>
+                    <field name="show_price"
+                           widget="boolean_fa_icon"
+                           options="{
+                                'show_in_line_section': true,
+                                'fa_icons': {
+                                    'icon_true': 'fa-check-circle',
+                                    'icon_false': 'fa-circle',
+                                },
+                                'terminology': {
+                                    'hover_true': 'Show prices',
+                                    'hover_false': 'Show prices',
+                                },
+                           }"
+                           attrs="{'invisible': ['|', ('display_type', '!=', 'line_section'), ('show_details', '!=', True)]}"/>
                     <field name="show_subtotal"
                            widget="boolean_fa_icon"
                            options="{
                                 'show_in_line_section': true,
                                 'fa_icons': {
-                                    'icon_true': 'fa-plus-circle',
-                                    'icon_false': 'fa-minus-circle',
+                                    'icon_true': 'fa-check-circle',
```

Comment:
Suggestion : fa-check-square

Reply:
Having different checkboxes icons didn't provide anything to the UX, that's why we're using the same icon for both features.

They can be easily identified by the Tooltip, and by their order :)

Reply:
I don't like having the same icon.

Reply:
Would `check-circle` / `check-square` be ok?

There aren't many choises to pick

Reply:
@ernestotejeda what do you think?

Reply:
I agree with @pedrobaeza, different icons are one more resource for the user to more easily and quickly identify the option they need to check/uncheck (or activate/deactivate). Once the user learns what functionality each icon corresponds to (even if it is not the most appropriate), they can locate it in a single glance without having to hover the mouse or learn the order in which they are located.

Reply:
Another thing that worries me is that I don't know if it's appropriate to add this functionality in this module. Perhaps the users who are using this module don't need this new functionality and it may be uncomfortable for them to have one more column that they don't use, or maybe they simply don't want one more.
Ping @pedrobaeza

Reply:
@ernestotejeda what about using:

- Show prices: `square` / `check-square`: https://fontawesome.com/v4.7.0/icon/check-square
- Show subtotals: `circle` / `check-circle`: https://fontawesome.com/v4.7.0/icon/check-circle

I don't think we'll find toggleable descriptive icons for these features :sweat_smile: 

The ones that were used before this PR for show subtotals were `plus-circle` and `minus-circle` but, honestly, using `check-circle` is more intuitive as users already know that if it's checked it means `true`

---

Regarding moving this to a new module.. I understand that users might not want this feature but having separate modules it probably overkill, don't you think? Specially because we depend on Show detail, and features are clearly related.

Maybe this could be managed by security groups, to enable / disable the features

Reply:
@ernestotejeda an update on this one?

Reply:
@ernestotejeda and @pedrobaeza : What do you think about that ?

Reply:
Works really fine, and user appreciate the option to show or hide what the need.
Nice job !
I'm waiting the PR validation to finish the translation on weblate (I did it for the actual version this afternoon).